### ISSUE TITLE
Question ouverte sur les types de dépôts

### ISSUE THREAD

**Issue by Julie-JDS** [Jardin-des-Sciences/website#4]
Dans le CCTP, apparaissait la recommandation suivante : 

2.2.4 Intégration dans le Linked Open Data (LOD)
À partir de ce riche socle de données, il est prévu d’intégrer plus avant Hist-Science dans le
Linked Open Data en proposant a minima :
● l’enrichissement des données par des bases tierces comme DBpedia
● une compatibilité avec le moissonnage du portail Isidore (voir les spécifications
http://www.huma-num.fr/sites/default/files/ressourcesdoc/guide_isidore_2012.pdf).

Est-ce que l'on répondra à cette préoccupation initiale avec les orientations prises ?
Est ce que ça complexifie les choses ?
Merci.
J.

**Comment by bzg** [comment]
Oui.  Pour la compatibilité avec le moissonnage du portail Isidore, le portail JDS exposera ses données sous le bon format.

Pour l'enrichissement des données par des bases tierces, c'est un enrichissement qui ne se fera pas directement dans notre base, mais dans la combinaison entre les contenus de notre base et des contenus associés, affichés conjointement depuis notre interface.

C'est un travail d'architecture de données qu'il reste à détailler.
